christfellowshipchurch/apollos-api
# Christmas Feature

### About
This PR adds a custom feature to the `/25days` page. The feature is an ActionList feature pointing to a specific content category. The custom features for the Christmas page and be managed the `christmas-features.yml` file.

### Test Instructions
* run this api branch and web locally
* go to `/25days` and scroll to bottom to see "25 Days of Hope" feature.

### Screenshots
![image](https://user-images.githubusercontent.com/46049974/142666316-a243f9dd-f356-4924-ae9a-548932e730d2.png)

### Closes Tickets
[CFDP-1826]


[CFDP-1826]: https://christfellowshipchurch.atlassian.net/browse/CFDP-1826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
